libflux/core/src/ast/flatbuffers/monotype.rs

```diff
@@ -40,7 +40,11 @@ fn build_base_node<'a>(
         builder.create_string(&s)
     });
     let errors = Some(builder.create_vector(errors.as_slice()));
-    fb::BaseNode::create(builder, &fb::BaseNodeArgs { loc, errors })
+    let comments = build_vec(base_node.comments, builder, |builder, s| {
```

Comment:
If you can, try to avoid using the same variable and then immediately overwriting it. This works fine but it might be better if you just inlined this.

Reply:
changes are done as suggested.